### PR TITLE
Added script for release license audit

### PR DESCRIPTION
**Pull request by cramja**
@hbdeshmukh Take a look, this will be useful in the future.

### CODE REVIEW

File: release/release_cmds.sh [hunk 1]
```diff
@@ -50,14 +51,14 @@ create_artifacts() {
   # Make the signature. This requires human input
   gpg -u $APACHE_USERNAME@apache.org --armor --output $PROJECT_NAME-$VERSION.tar.gz.asc --detach-sign $PROJECT_NAME-$VERSION.tar.gz
   # Make hashes
+  for algo in 256 512; do
+    shasum -a $algo $PROJECT_NAME-$VERSION.tar.gz > $PROJECT_NAME-$VERSION.tar.gz.sha$algo
```
Comment (cramja) [inline release/release_cmds.sh]: @hbdeshmukh PS, I added these changes in this PR as well. We now will use un-cracked versions of SHA 😄

File: release/release_cmds.sh [hunk 2]
```diff
@@ -100,6 +101,23 @@ publish_candidate() {
...
+    curl http://apache.spinellicreations.com//creadur/apache-rat-0.12/apache-rat-0.12-bin.tar.bz2 > rat.tar.bz2
+    tar -xzf rat.tar.bz2
+    rm rat.tar.bz2
+  fi
+  java -jar apache-rat-0.12/apache-rat-0.12.jar -e release -e build -e cmake* -d .. > rat-report.txt
+  if [ $? == 0 ] ; then
+     echo "created rat-report.txt"
+  else
+     echo "error running rat"
+  fi
```
Comment (hbdeshmukh) [inline release/release_cmds.sh]: Do you think we should delete the rat source code after we are done using it?

**Comment by hbdeshmukh** [comment]
Looks good to me, the audit feature should be really useful.

File: release/release_cmds.sh (hunk 2 above)
Comment (cramja) [inline release/release_cmds.sh]: It's a jar - like a java executable. I think it's fine to keep around seeing as gitignore already ignores it.

**Comment by hbdeshmukh** [comment]
Hi @cramja your fork is behind the master. Can you please sync it? Then I can merge.

**Comment by cramja** [comment]
Sure, also I merged while doing that. I think it's kosher so long as someone looked over the PR w/ a +1.

**Pull request merged**